rosbag2_storage_default_plugins/test/rosbag2_storage_default_plugins/sqlite/storage_test_fixture.hpp

```diff
@@ -113,8 +115,11 @@ class StorageTestFixture : public TemporaryDirectoryFixture
   {
     std::unique_ptr<rosbag2_storage::storage_interfaces::ReadOnlyInterface> readable_storage =
       std::make_unique<rosbag2_storage_plugins::SqliteStorage>();
+
+    auto db_file = rosbag2_storage::FilesystemHelper::concat({temporary_dir_path_, "rosbag"});
+
     readable_storage->open(
-      temporary_dir_path_, rosbag2_storage::storage_interfaces::IOFlag::READ_ONLY);
+      db_file, rosbag2_storage::storage_interfaces::IOFlag::READ_ONLY);
```

Comment:
with the changes introduced in this PR, I think this `db_file` should point to a existing `.db3` file